tests/runtests.py

```diff
@@ -58,15 +58,17 @@ def main() -> None:
         choices=("3.9", "3.10", "3.11", "3.12", "3.13"),
         help="Target Python version for the test (default: %(default)s).",
     )
-    parser.add_argument("path", help="Path of the stub to test in format <folder>/<stub>, from the root of the project.")
+    parser.add_argument(
+        "path", help=f"Path of the stub to test in format {PurePath('<folder>', '<stub>')}, from the root of the project."
+    )
     args = parser.parse_args()
     path: str = args.path
     run_stubtest: bool = args.run_stubtest
     python_version: str = args.python_version
 
     path_tokens = Path(path).parts
     if len(path_tokens) != 2:
-        parser.error("'path' argument should be in format <folder>/<stub>.")
+        parser.error(f"'path' argument should be in format {PurePath('<folder>', '<stub>')}.")
```

Comment:
Isn't `Path` automatically subclassed? (in Windows, it's a `WindowsPath`, which stringifies the way you want)
```suggestion
        parser.error(f"'path' argument should be in format {Path('<folder>', '<stub>')}.")
```

```py
>>> from pathlib import Path
>>> path = Path('<folder>', '<stub>')
>>> print(path, type(path))
<folder>\<stub> <class 'pathlib._local.WindowsPath'>
```

I still would prefer fixing the support for forward-slashes. As that's *intended* to work.